src/argilla/server/policies.py

```diff
@@ -379,6 +390,7 @@ async def is_allowed(actor: User) -> bool:
                 actor, metadata_property.dataset.workspace_id
             )
 
+            # TODO: Check if we need to change this to support allowed_roles
```

Comment:
Yes, the metrics should apply the same policy logic as for the `get` action.

Reply:
Then I guess we can remove this policy and use simply `get` policy?

Reply:
Done, now using `MetadataPropertyPolicyV1.get` instead.